### DISCUSSION TITLE
Can I use any npm script in kit?

### DISCUSSION THREAD

**Discussion by arunmohantp** [johnlindquist/kit#251] [Q&A]
I used it like below and it's not working 

let http =  require('https');

var options = {
    host: 'myUrl',
    path: '/flights/listings?segment1.origin=ORD&segment1.destination=ATL&segment1.departureDate=2021-07-10&adult=1&segment2.origin=ATL&segment2.destination=ORD&segment2.departureDate=2021-07-28&links=AD&cabinClass=economy,
      headers: {'Authorization': '***',
    'Key':'***',
    'Accept':****'}
};

callback = function(response) {
    var str = ''
    response.on('data', function (chunk) {
        str += chunk;
    });

    response.on('end', function () {
        console.log(str);
    });
}

var req = http.request(options, callback);
req.end();

**Comment by johnlindquist** [comment]
Try
```js
let http = await import('https');
```

Or
```js
import http from 'https'
```

These scripts don't support "require" since they're "modules" and follow ESM rules.
